src/adapters/data_models/shortcuts_model/shortcuts_proxy_model.cpp

```diff
@@ -58,6 +64,10 @@ bool ShortcutsProxyModel::filterAcceptsRow(
 void ShortcutsProxyModel::setFilterString(QString newFilterString)
 {
     m_filterString = newFilterString;
+    if(newFilterString.isEmpty())
+        m_filterScorer = std::nullopt;
+    else
+        m_filterScorer = rapidfuzz::fuzz::CachedRatio(m_filterString.toUcs4());
```

Comment:
I think you probably want `toUcs4` instead of `toString`. `toString` returns a utf8 string and rapidfuzz would just match each element in there. This works for ascii, but not for many other languages.